packages/message-manager/src/utils.ts

```diff
@@ -132,3 +133,18 @@ export function validateEncryptionPublicKeyMessageData(
     throw new Error(`Invalid "from" address: ${from} must be a valid string.`);
   }
 }
+
+/**
+ * Validates messageData for the eth_decrypt message and throws in
+ * the event of any validation error.
+ *
+ * @param messageData - address string to validate.
+ */
+export function validateDecryptedMessageData(
+  messageData: DecryptMessageParams,
+) {
+  const { from } = messageData;
+  if (!from || typeof from !== 'string' || !isValidHexAddress(from)) {
```

Comment:
Rather than duplicate this, can we create a private but generic address function that both of the validate methods call?

Reply:
Fixed it for this and the other related validation functions